doc/examples.xml

```diff
@@ -309,3 +309,44 @@ gap> GeneralisedPetersenGraph(IsMutableDigraph, 9, 4);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="KnightsGraph">
+<ManSection>
+  <Oper Name="KnightsGraph" Arg="[filt, ]n, k"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>n</A> and <A>k</A> are both positive integers, then this operation returns
+    the <E>Knight's Graph</E> for a n by k board. <P/>
+
+    From
+    <URL>https://en.wikipedia.org/wiki/Knight%27s_graph</URL>:
+    <P/>
+
+    <Q>The knights graph for a given n by k board size consists all of n * k vertices, where
```

Comment:
This formulation makes me think that you're giving a quotation (which is what `<Q>` tags mean, I think), but it's not a quote! Perhaps you could do something like:

As explained on {URL}, the knight's graph... etc